presto-main/src/main/java/com/facebook/presto/metadata/TableHandle.java

```diff
@@ -28,7 +27,7 @@
 
 public final class TableHandle
 {
-    private final ConnectorId connectorId;
+    private final String catalog;
```

Comment:
Why don't just move the `ConnectorId` to the SPI? Usually we are trying to avoid bare types for any kind of ids. e.g.: PlanNodeId, StageId, etc - are all wrappers.